csatar-plugins/csatar/controllers/scouts/create.htm

```diff
@@ -11,6 +11,7 @@
 
         <div class="layout-row">
             <?= $this->formRender() ?>
+            <?= $this->relationRender('allergies') ?>
```

Comment:
Please add relation render in new 'layout-row' div, otherwise it will overlap with form buttons.
```suggestion
        <div class="layout-row">
            <?= $this->formRender() ?>
        </div>

        <div class="layout-row">
            <?= $this->relationRender('allergies') ?>
        </div>
```